src/WidgetApi.ts

```diff
@@ -363,11 +363,14 @@ export class WidgetApi extends EventEmitter {
 
     public readRoomEvents(
         eventType: string,
-        limit = 25,
+        limit?: number,
         msgtype?: string,
         roomIds?: (string | Symbols.AnyRoom)[],
     ): Promise<unknown> {
-        const data: IReadEventFromWidgetRequestData = {type: eventType, msgtype: msgtype, limit};
+        const data: IReadEventFromWidgetRequestData = {type: eventType, msgtype: msgtype};
+        if (limit !== undefined) {
+            data.limit = limit;
+        }
```

Comment:
Is this bit neccessary?

Reply:
The MSC could also be updated to mention that `undefined` is equivalent to not being present.